.ci/riscv-tests.sh

```diff
@@ -7,20 +7,23 @@ check_platform
 mkdir -p toolchain
 
 # GNU Toolchain for RISC-V
-GCC_VER=12.2.0-1
-XPACK_REPO=https://github.com/xpack-dev-tools/riscv-none-elf-gcc-xpack/releases/download
+GCC_VER=2023.01.04-nightly
```

Comment:
What if GCC version is not changed?

Reply:
By default, RISCOF uses [riscv-gnu-toolchain](https://github.com/riscv-collab/riscv-gnu-toolchain) to compile the riscv-arch-test in `riscof_rv32emu.py` and `riscof_sail_cSim.py`. For example in `riscof_rv32emu.py`:
```py
self.compile_cmd = 'riscv{1}-unknown-elf-gcc -march={0} \
        -static -mcmodel=medany -fvisibility=hidden -nostdlib -nostartfiles -g\
        -T '+self.pluginpath+'/env/link.ld\
        -I '+self.pluginpath+'/env/\
        -I ' + archtest_env + ' {2} -o {3} {4}'
```
Therefore, I think modifying both of the python files may avoid to change the gcc version.

Reply:
Got it. Let's stick to `riscv-gnu-toolchain`. Git commit messages should be amended for the above consideration.